be/src/exec/vectorized/hdfs_scanner.h

```diff
@@ -118,6 +118,8 @@ struct HdfsScannerParams {
     HdfsScanProfile* profile = nullptr;
 
     std::atomic<int32_t>* open_limit;
+
+    std::vector<const TIcebergDeleteFile*> deletes;
```

Comment:
This field is only used in hdfs_scanner_orc right?

Reply:
yes

Reply:
Well, if so perhaps define a orc specific struct is more proper. such as `HdfsOrcScannerParams`